--- gmrun-0.9.1/src/gtkcompletionline.cc	2003-06-22 01:14:34.000000000 +0200
+++ gmrun-0.9.1/src/gtkcompletionline.cc	2007-08-01 00:32:35.000000000 +0200
@@ -225,12 +225,9 @@
   const char* i = str.c_str();
   while (*i) {
     char c = *i++;
-    switch (c) {
-     case ' ':
-      res += '\\';
-     default:
-      res += c;
-    }
+    if (c == ' ' || c == '(' || c == ')' || c =='\'')
+        res += '\\';
+    res += c;
   }
   return res;
 }

