--- gmrun-0.9.1/src/prefs.cc~   2007-04-01 19:59:43.000000000 +0000
+++ gmrun-0.9.1/src/prefs.cc    2007-04-01 19:59:51.000000000 +0000
@@ -10,6 +10,7 @@
  *****************************************************************************/


+#include <cstdlib>
 #include <fstream>
 #include <iostream>
 #include <stdio.h>
--- a/src/ci_string.h
+++ b/src/ci_string.h
@@ -8,6 +8,7 @@
 
 #include <string>
 #include <ctype.h>
+#include <cstring>
 
 struct ci_char_traits : public std::char_traits<char>
 {
--- a/src/gtkcompletionline.cc
+++ b/src/gtkcompletionline.cc
@@ -30,6 +30,7 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <cstring>
 using namespace std;
 
 #include "gtkcompletionline.h"
